silq/analysis/analysis.py

```diff
@@ -715,8 +717,44 @@ def analyse_EPR(empty_traces: np.ndarray,
             'mean_high_blip_duration': results_read['mean_high_blip_duration']}
 
 
-def analyse_flips(up_proportions_arrs: List[np.ndarray],
-                  threshold_up_proportion: Union[Sequence, float]):
+def analyse_threshold_up_proportion(up_proportions_arrs: np.ndarray,
+                                    shots_per_frequency: int):
```

Comment:
Missing function docstring. I think it's especially needed in this case because the algorithm contains several steps and is note trivial